src/Symfony/Component/Form/Form.php

```diff
@@ -206,19 +231,19 @@ public function getConfig()
      */
     public function getName()
     {
-        return $this->config->getName();
+        return $this->name;
     }
 
     /**
      * {@inheritdoc}
      */
     public function getPropertyPath()
     {
-        if (null !== $this->config->getPropertyPath()) {
-            return $this->config->getPropertyPath();
+        if ($this->propertyPath || $this->propertyPath = $this->config->getPropertyPath()) {
```

Comment:
just wondering: can it happen that the properypath on the config changes? since now we are "caching" it we would not reflect those config changes when calling the getter multiple times

Reply:
ok as discussed the config is locked :blush: